signal_id/mask.py

```diff
@@ -232,10 +236,12 @@ def independent_cubes(self, struct=None):
     # -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 
     def enable_backup(self):
-        self._implict_backup = True
+        self.is_backup_enabled = True
+        return self
 
     def disable_backup(self):
-        self._implict_backup = False
+        self.is_backup_enabled = False
+        return self
```

Comment:
why is `self` being returned here?  Just curious what the interface is meant to do.  Normally a state change like this would have no return, right?  You might also make `is_backup_enabled` a `property` since it's so simple